README.md

```diff
@@ -25,60 +34,55 @@ Kuzzle enables you to build modern web applications and complex IoT networks in
 The easiest way to setup a kuzzle server for Linux-like systems without prerequisites is to download and run our installation script:
 
 ```bash
-$ sudo bash -c "$(curl http://get.kuzzle.io/)"
+$ sudo bash -c "$(curl https://get.kuzzle.io/)"
 ```
 
-You can get detailed information about how to [start kuzzle with docker on docs.kuzzle.io](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/#docker)
+You can get detailed information about how to [start kuzzle with docker on docs.kuzzle.io](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/#docker)
 
 ### Manual install
 
-Check our [complete installation guide on docs.kuzzle.io](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/#manually)
+Check our [complete installation guide on docs.kuzzle.io](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/#manual-installation)
 
 ## Quick start with Kuzzle
 
-* [Install and start Kuzzle server](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/)
-* [Choose a SDK](https://docs.kuzzle.io/sdk-reference/essentials/)
+* [Install and start Kuzzle server](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle)
+* [Choose a SDK](https://docs.kuzzle.io/sdk-reference/)
 * Build your application without caring about your backend !
 
-Check the [**Getting started page on docs.kuzzle.io**](https://docs.kuzzle.io/guide/getting-started/)
+Check the [**Getting started page on docs.kuzzle.io**](https://docs.kuzzle.io/guide/1/getting-started/first-steps/)
 
-### NodeJS Sample
+### Node.js samples
 
 ```bash
 npm install kuzzle-sdk
 ```
 
 ```javascript
-const
-    Kuzzle = require('kuzzle-sdk'),
-    kuzzle = new Kuzzle('http://localhost:7512')
-
-const filter = {
-    exists: {
-        field: 'message'
-    }
-}
-
-// Subscribe to data changes in an app
-kuzzle
-    .collection('mycollection', 'myindex')
-    .subscribe(filter, function(error, result) {
-        // triggered each time a document is updated !
-        console.log('message received from kuzzle:', result)
-    })
-
-// Creating a document from another app will notify all subscribers
-kuzzle
-    .collection('mycollection', 'myindex')
-    .createDocument(document)
+const 
+  {
+    Kuzzle,
+    WebSocket
+  } = require('kuzzle-sdk');
+
+// Connect the SDK to your Kuzzle server with WebSocket protocol
+const kuzzle = new Kuzzle(
+  new WebSocket('localhost')
+);
+await kuzzle.connect();
+
+// Subscribes to database changes
+await kuzzle.realtime.subscribe('my-index', 'my-collection', {}, msg => {
+  console.log('Realtime notification received from Kuzzle:', msg);
+});
 ```
 
 ### Useful links
 
 * [Full documentation](https://docs.kuzzle.io/)
-* [SDK Reference](https://docs.kuzzle.io/sdk-reference/essentials/)
-* [API Documentation](https://docs.kuzzle.io/api-documentation/connecting-to-kuzzle/)  
-* [Data Validation Documentation](https://docs.kuzzle.io/validation-reference/schema/)
+* [SDKs Reference](https://docs.kuzzle.io/sdk-reference/)
```

Comment:
To me, the very first link of that list should be something like this: `[How to run a Kuzzle instance](https://docs.kuzzle.io/guide/1/getting-started/running-kuzzle/)`

Followed by that one: `[Basic usage examples](https://docs.kuzzle.io/guide/1/getting-started/first-steps/)`

And then the list that you amended, which otherwise seems perfect :+1: